sdk/python/kfp/dsl/_container_op.py

```diff
@@ -999,6 +1004,25 @@ def set_display_name(self, name: str):
   def __repr__(self):
     return str({self.__class__.__name__: self.__dict__})
 
+  def add_cpu_request(self, cpu: str):
```

Comment:
Don't these belong to the `Container` class?

Reply:
It is a bit unclear to me what we have in the BaseOp vs the ContainerOp? I have moved it.